README.md

```diff
@@ -1,6 +1,6 @@
 # Complexity & Travis-CI Lab
 
-In this lab, you'll be implementing simple algorithms for checking basic properties of code complexity, and working with Travis-CI for Continuous Integration.
+In this lab, you'll be implementing simple algorithms for checking basic properties of code complexity.
```

Comment:
This is a good change but we should also mention Travis-CI in a separate line

Reply:
I added the change!